lib/node_modules/@stdlib/stats/base/meanpn/README.md

```diff
@@ -67,9 +67,9 @@ The function has the following parameters:
 
 -   **N**: number of indexed elements.
 -   **x**: input [`Array`][mdn-array] or [`typed array`][mdn-typed-array].
--   **stride**: index increment for `x`.
+-   **strideX**: stride length for `x`.
 
-The `N` and `stride` parameters determine which elements in `x` are accessed at runtime. For example, to compute the [arithmetic mean][arithmetic-mean] of every other element in `x`,
+The `N` and stride parameters determine which elements in the strided array are accessed at runtime. For example, to compute the [arithmetic mean][arithmetic-mean] of every other element in the input array,
 
 ```javascript
 var floor = require( '@stdlib/math/base/special/floor' );
```

Comment:
@dhruvilmehta Would you also mind updating the examples throughout this package to no longer use `floor`?